### PR TITLE
Enable score plot download

### LINKED ISSUE #386: Score plot download
This was lost in the conversion to C3 in #219.

### PR DESCRIPTION
**Pull request by waxlamp**
Closes #386

### CODE REVIEW

File: web/src/utils/exporter.js [hunk 1]
```diff
@@ -65,6 +66,36 @@ export function svg2url(svgElement, options = {}) {
     copy.insertAdjacentHTML('afterbegin', `<style>${rules.join('\n')}</style>`);
   }
 
+  // Need some special treatment for C3 charts.
```
Comment (sgratzl) [inline web/src/utils/exporter.js]: can you please extract this custom logic to a separate function

Comment (waxlamp) [reply web/src/utils/exporter.js]: Let me know if edc63cd looks ok.

File: web/src/utils/exporter.js [hunk 2]
```diff
@@ -3,15 +3,50 @@ import font from '!url-loader?limit=undefined!@openfonts/barlow-condensed_all/fi
...
+      stroke: black;
+    }
+
+    .tick line {
+      stroke: black;
+    }
+  </style>`);
+
+  // For some reason, C3 uses the "color" style property instead of "fill".
+  select(node)
```
Comment (sgratzl) [inline web/src/utils/exporter.js]: btw. a pure DOM version would be
```js
Array.from(node.querySelectorAll('circle')).forEach((circle) => {
   circle.style.fill = circle.style.color;
   circle.style.color = null;
});
```

Comment (waxlamp) [reply web/src/utils/exporter.js]: I opened #413 to address this.

**Pull request merged**